packages/themes/pennwell/styles/components/_item.scss

```diff
@@ -320,4 +320,19 @@
     }
   }
 
+  // Magazine
+  &__magazine-issue-link,
```

Comment:
To make both magazine issue card buttons (e.g. https://www.waterworld.com/magazine/468041) and magazine issue item buttons/links (e.g. https://www.waterworld.com/magazine/5ca26a8b08cf3add038b45b7) consistent, instead of making new styles, just use the buttons from `<endeavor-magazine-issue-card>` and apply them to `<endeavor-magazine-issue-item>`

In other words, move the subscribe and digital edition URLs from: https://github.com/endeavorb2b/websites/blob/e6bcf1cc6420e49f4d80bd23fb0daa931e245958/packages/common/components/magazine-issue/card.marko#L40

And replace the ones found here:
https://github.com/endeavorb2b/websites/blob/e6bcf1cc6420e49f4d80bd23fb0daa931e245958/packages/common/components/magazine-issue/item.marko#L47

Once that's complete, the new styles can be removed.

Reply:
added class="btn item__button" to elements and removed the custom styling